sequencesender/sequencesender.go

```diff
@@ -126,7 +169,7 @@ func (s *SequenceSender) tryToSendSequence(ctx context.Context) {
 		return
 	}
 
-	lastVirtualBatchNum, err := s.state.GetLastVirtualBatchNum(ctx, nil)
+	lastVirtualBatchNum, err := s.etherman.GetLatestBatchNumber()
```

Comment:
It looks like this call is the same as line 131. Maybe we can avoid calling the SC again.